### PR TITLE
Update spotify-controls extension

### PR DESCRIPTION
**Pull request by stevensd2m**
## Description

Add a command to play Discover Weekly playlist

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/spotify-controls/.gitignore
Comment (mil3na) [inline extensions/spotify-controls/.gitignore:1]: This file should not be needed.

**Review by mil3na — CHANGES_REQUESTED** [review]
Hi! 
Thanks for your contribution! My name is Milena and I will be reviewing your changes today.
I feel like adding Discovery Weekly (or playing any playlist for that matter) is a bit out of scope for a commands extension. I think the goal here is to control Spotify without adding anything specific. 
Maybe something that would make more sense on the spotify-player extension?

**Comment by stevensd2m** [comment]
@mil3na thanks for the review 😄  given your feedback I'll close the PR then and open it on the player instead

**Comment by pernielsentikaer** [comment]
Closing as requested

**Pull request closed without merging**